store/schema-update-playbook.md

```diff
@@ -0,0 +1,26 @@
+# Bytebase Schema Update Playbook
+
+All schemas are located at [store/migration](https://github.com/bytebase/bytebase/tree/main/store/migration). Bytebase schema release follows semantic versioning. This document goes over the scheme update process during development and release.
+
+## Development
+We use different schema versions for [dev](https://github.com/bytebase/bytebase/blob/main/bin/server/cmd/profile_dev.go) and [release](https://github.com/bytebase/bytebase/blob/main/bin/server/cmd/profile_release.go). 
+
+If version numbers are the same,
+1. Create a new directory under [migration directory](https://github.com/bytebase/bytebase/tree/main/store/migration) with the new version, e.g. 1.3.0.
+2. Copy over the [latest.sql](https://github.com/bytebase/bytebase/blob/main/store/migration/1.1.0/latest.sql) schema file and [latest_data.sql](https://github.com/bytebase/bytebase/blob/main/store/migration/1.1.0/latest_data.sql) data filefrom the previous version. And make a PR for changes above.
```

Comment:
```suggestion
2. Copy over the [latest.sql](https://github.com/bytebase/bytebase/blob/main/store/migration/1.1.0/latest.sql) schema file and [latest_data.sql](https://github.com/bytebase/bytebase/blob/main/store/migration/1.1.0/latest_data.sql) data file from the previous version. And make a PR for changes above.
```

Reply:
Here we always point to the 1.1.0 which may cause confusion

Reply:
Yeah, let's starting from 1.0.0.